### PR TITLE
Move event export to src/Module/Events/Export

### PR DESCRIPTION
**Pull request by nupplaphil**
Part of #4090

Step by Step, I do want to move the events :-)

Tested it locally and fixed ical as well ;)

### CODE REVIEW

File: src/Module/Events/Export.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Events;
```
Comment (MrPetovan) [inline src/Module/Events/Export.php]: I am not happy that the namespace is plural when only one other is, do you mind replacing with `Calendar`, including the `Events\Json` Module class?

File: src/Module/Events/Export.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+			} else {
+				$returnPath = 'events/' . $this->parameters['nickname'];
+			}
+
+			$this->baseUrl->redirect($returnPath);
+		}
+
+		// If nothing went wrong we can echo the export content
+		if ($evexport["success"]) {
+			$this->response->setHeader(sprintf('content-disposition: attachment; filename="%s-%s.%s"',
```
Comment (MrPetovan) [inline src/Module/Events/Export.php]:
```suggestion
			$this->response->setHeader(sprintf('Content-Disposition: attachment; filename="%s-%s.%s"',
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Module/Events/Export.php (hunk 1 above)
Comment (nupplaphil) [reply src/Module/Events/Export.php]: Hehe, I was at the same point yesterday, but I thought it's maybe intended that it's called events and not calendar.

Comment (MrPetovan) [reply src/Module/Events/Export.php]: I don't think so. There were two files in the `mod/` folder regarding the calendar feature called `cal.php` and `events.php` but we aren't limited by the flat structure anymore.

File: static/routes.config.php [hunk 3]
```diff
@@ -365,6 +365,9 @@
 
 	'/bookmarklet'         => [Module\Bookmarklet::class,  [R::GET]],
 
+	'/calendar/{nickname}/export[/{format}]' => [Module\Calendar\Export::class, [R::GET]],
+	'/calendar/json'                           => [Module\Calendar\Json::class, [R::GET]],
```
Comment (MrPetovan) [inline static/routes.config.php]: Better alignment.
```suggestion
	'/calendar/json'                         => [Module\Calendar\Json::class,   [R::GET]],
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by MrPetovan** [comment]
Standards and messages.po checks need to be addressed.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**